web/cgi-bin/horas/specials.pl

```diff
@@ -16,8 +16,11 @@ sub specials {
   my $lang = shift;		 
   
   $octavam = ''; #check duplicate commemorations
-  my %w = (columnsel($lang)) ? %winner : %winner2; 
-  
+  my %w = (columnsel($lang)) ? %winner : %winner2;
+
+  my $hymntrans = $translate{$lang}{Hymnus};
```

Comment:
Maybe this should be

    my $hymntrans = 'Hymnus' || $translate{$lang}{Hymnus};

This is supposing that `$hymntrans` is used again later on (which I imagine that it is, given the last hunk of the patch).